### ISSUE TITLE
Does not compile on 64-bit MSVC

### ISSUE THREAD

**Issue by jessiepathfinder** [RobertDurfee/Disassembler#4]

**Comment by RobertDurfee** [comment]
Hey, @jessielesbian! Thanks so much for your interest in this project! Unfortunately it has been a few years since I have looked at this project--let alone compiled it--so I'm afraid I won't be much help.

Last I remember, I was using a x86_64 gcc compiler on Linux, but original development (even more years ago) was actually using MSVC. So there shouldn't be any inherent reason why it can't compile, perhaps after fixing some compiler errors.

Also, just a heads up, this was one of my first projects, so the design is a bit questionable and support is limited. Bugs are likely everywhere. Don't even think about trying to disassemble any instruction set extensions! I mainly leave this up for nostalgia and educational purposes, along with many of my older projects.

Thanks again for your interest! I do have rather limited time, but I will try to help with any further questions you have.

**Comment by RobertDurfee** [comment]
I do not remember doing this, but it looks like I made a tiny `Makefile` for an example in the `Example` directory [here](https://github.com/RobertDurfee/Disassembler/blob/master/Example/Makefile). Perhaps this can shed some light on what needs to be compiled and how? 

It does seem I didn't want to compile for 64-bit (given by `-m32` in the `Makefile`), but I don't remember why... It's possible I made some assumptions about machine word sizes? Also, notice the `-Wno-write-strings`, which might be needed depending on your setup treats warnings.
